Specialization/Blood.lua

```diff
@@ -124,18 +165,24 @@ function DeathKnight:BloodGlowCooldowns()
 	local talents = fd.talents;
 	local covenantId = fd.covenant.covenantId;
 
-	local abominationLimbReady = DeathKnight.db.abominationLimbAsCooldown and covenantId == Necrolord and cooldown[BL.AbominationLimb].ready;
-	local dancingRuneWeaponReady = DeathKnight.db.bloodDancingRuneWeaponAsCooldown and cooldown[BL.DancingRuneWeapon].ready;
+	local abominationLimbReady = covenantId == Necrolord and cooldown[COMMON.AbominationLimb].ready;
+	local abominationLimbTalentReady = talents[COMMON.AbominationLimbTalent] and cooldown[COMMON.AbominationLimbTalent].ready;
+	local dancingRuneWeaponReady = talents[BL.DancingRuneWeapon] and cooldown[BL.DancingRuneWeapon].ready;
+	local boneStormReady = talents[BL.Bonestorm] and cooldown[BL.Bonestorm].ready;
+	local empowerRuneweaponReady = talents[COMMON.EmpowerRuneWeapon] and cooldown[COMMON.EmpowerRuneWeapon].ready;
 
 	if DeathKnight.db.alwaysGlowCooldowns then
-		MaxDps:GlowCooldown(BL.AbominationLimb, abominationLimbReady);
+		MaxDps:GlowCooldown(COMMON.AbominationLimb, abominationLimbReady);
+		MaxDps:GlowCooldown(COMMON.AbominationLimbTalent, abominationLimbTalentReady);
+		MaxDps:GlowCooldown(BL.Bonestorm, boneStormReady);
+		MaxDps:GlowCooldown(COMMON.EmpowerRuneWeapon, empowerRuneweaponReady);
 		MaxDps:GlowCooldown(BL.DancingRuneWeapon, dancingRuneWeaponReady);
 	else
-		local abominationLimbCooldownTrigger = abominationLimbReady and not buff[BL.DancingRuneWeapon].up;
-		local dancingRuneWeaponCooldownTrigger = dancingRuneWeaponReady and (not talents[BL.Blooddrinker] or not cooldown[BL.Blooddrinker].ready)
-
-		MaxDps:GlowCooldown(BL.AbominationLimb, abominationLimbCooldownTrigger);
-		MaxDps:GlowCooldown(BL.DancingRuneWeapon, dancingRuneWeaponCooldownTrigger);
+		MaxDps:GlowCooldown(BL.Bonestorm, boneStormReady);
```

Comment:
Same as above

Reply:
same response. I see no reason not to add a glow to it like we do other optional cooldowns.